fg_lib/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _fg_lib:
+
+Nordic Fuel Gauge library (FG lib)
```

Comment:
You're inconsistent in capitalization and name usage throughout the doc. I've seen the following names for the lib in the doc and API docs:
- Nordic Fuel Gauge library
- the fuel gauge library
- Fuel Gauge
- the battery fuel gauge library

I suggest we stick to one name, and possibly a short one. How about the "nRF Fuel Gauge" @lemrey suggested?

Reply:
Yep, I agree with this. Working on renaming everything to nRF Fuel Gauge (and prefix nrf_fuel_gauge instead of fg_lib everywhere)